### DISCUSSION TITLE
How to configure locale for non-Chinese text?

### DISCUSSION THREAD

**Discussion by sbue** [ant-design/ant-design-mobile#4919] [Q&A] [answered]
Hi all,

I want to change the locale so it supports Spanish. For example, I want to change "locale.Form.required" to the localized, Spanish word: https://github.com/ant-design/ant-design-mobile/blob/master/src/components/form/form-item.tsx

**Comment by awmleer** [comment, marked as answer]
You can wrap your app with a `ConfigProvider` to change the locale. For example:

```jsx
<ConfigProvider locale={enUS}>
  <App />
</ConfigProvider>
```

However, currently there are only zh-CN and en-US locales. But you can write a locale object by yourself and pass it to `ConfigProvider`. it is not hard at all, just refer to the base locale [here](https://github.com/ant-design/ant-design-mobile/blob/master/src/locales/base.ts). And of course, you can either write the locale in your own project or open a pull request to add a Spanish locale to antd-mobile.
